### ISSUE TITLE
ESM support

### ISSUE THREAD

**Issue by philkunz** [anbraten/gitea-js#7]
types are not being found when package is used in ESM context.

**Comment by anbraten** [comment]
Maybe related to  🤔 

https://publint.dev/gitea-js@1.19.1
https://arethetypeswrong.github.io/?p=gitea-js%401.19.1

**Comment by philkunz** [comment]
It should look like something like this:

```
"exports": {
        ".": {
            // Entry-point for `import "my-package"` in ESM
            "import": {
                // Where TypeScript will look.
                "types": "./types/esm/index.d.ts",
                // Where Node.js will look.
                "default": "./esm/index.js"
            },
            // Entry-point for `require("my-package")` in CJS
            "require": {
                // Where TypeScript will look.
                "types": "./types/commonjs/index.d.cts",
                // Where Node.js will look.
                "default": "./commonjs/index.cjs"
            },
        }
    },
```

**Comment by anbraten** [comment]
Just released new versions for gitea 1.20 and 1.21 which should include the fixed types. Thanks for reporting
